// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

syntax = "proto2";

message t_dln_stats_item {
   required uint32 count = 1;
   required uint32 total = 2;
   required uint32 min = 3;
   required uint32 max = 4;
}

message t_dln_stats {
   required uint32 cu_ue_index = 1;
   required uint32 is_srb = 2;
   required uint32 rb_id = 3;
   required t_dln_stats_item sdu_bytes = 4;
   required t_dln_stats_item window = 5;
}

message dl_north_stats {
   required uint64 timestamp = 1;
   repeated t_dln_stats stats = 2;
}





